tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.0"
+__version__ = "17.0.0"
```

Comment:
This is the version for the plugin right? So this should be 0.1.1, no?

Reply:
To keep things simple, plugins are often versioned to match the Tutor version that they support.